src/main/java/seedu/duke/command/DeleteCommand.java

```diff
@@ -23,40 +24,60 @@ public DeleteCommand(String listType, String command) {
         this.command = command;
     }
 
-    public static Command parse(String input) throws WrongNumberOfArgumentsException, WrongNumberFormatException {
-        String[] inputParameters = input.trim().split(" ", 2);
+    /**
+     * Parses the user input and returns the corresponding DeleteCommand.
+     *
+     * @param input the processed user input with the command keyword removed.
+     * @return the DeleteCommand corresponding to the list type and event indicated in the input.
+     * @throws MissingSemicolonException if the input does not contain any semicolons to separate input fields.
+     * @throws WrongNumberOfArgumentsException if the event type or event index is missing.
+     * @throws WrongNumberFormatException if the event index is not an integer.
+     */
+    public static Command parse(String input) throws MissingSemicolonException, WrongNumberOfArgumentsException, WrongNumberFormatException {
+        if (!input.contains(";")) {
```

Comment:
Nice. I think this is a good feature to add to remind users about the format to use